relay-server/src/actors/project_buffer.rs

```diff
@@ -232,6 +234,10 @@ impl BufferService {
                 // 3. Disk I/O operations tends to be more sequential using WAL.
                 // 4. WAL uses many fewer fsync() operations and is thus less vulnerable to problems on systems where the fsync() system call is broken.
                 .journal_mode(SqliteJournalMode::Wal)
+                // WAL mode is safe from corruption with synchronous=NORMAL.
+                // When synchronous is NORMAL, the SQLite database engine will still sync at the most critical moments, but less often than in FULL mode.
+                // Which guarantees good balance between safety and speed.
+                .synchronous(SqliteSynchronous::Normal)
```

Comment:
I also find the following relevant, from the [docs](https://www.sqlite.org/pragma.html#:~:text=A%20transaction%20committed%20in%20WAL%20mode%20with%20synchronous%3DNORMAL%20might%20roll%20back%20following%20a%20power%20loss%20or%20system%20crash.):

> A transaction committed in WAL mode with synchronous=NORMAL might roll back following a power loss or system crash.